Rivers/Analysis/Traversal/DepthFirstTraversal.cs

```diff
@@ -20,13 +20,13 @@ public void Run(Node entrypoint)
                 throw new ArgumentNullException(nameof(entrypoint));
             
             var visited = new HashSet<Node>();
-            var stack = new Stack<(Node node, Edge edge)>();
-            stack.Push((entrypoint, null));
+            var stack = new Stack<(Node node, Edge edge, int Depth)>();
```

Comment:
Change `Depth` to `depth` for name consistency.

Reply:
Done.